pkgs/applications/graphics/unigine_heaven/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib,
+stdenv, fetchurl
+
+###
+, makeWrapper
+
+###
+, libX11,
+libXext,
+libXrandr,
+freetype,
+fontconfig,
+libXrender,
+libXinerama, autoPatchelfHook, libGL,
```

Comment:
Please stay consistent with one parameter per line. Also the convention is to have the comma at the start of each line. This keeps diffs cleaner when new parameters are added at the end of the list as the diff does not need to change the previous line just to add a comma.

i.e.:
```
{
, lib
, stdenv
, fetchurl
```
and so on